README.md

```diff
@@ -47,6 +47,7 @@ For previous versions, we only provide the version requirements of the `all` pac
 
 | CIF Connector | AEM 6.4 | AEM 6.5 | Magento | Java |
 |---------------|---------|---------|---------|------|
+| 1.2.0         | 6.4.4.0 | 6.5.0   | 2.3.4 & 2.3.5  | 1.8  |
 | 1.0.0         | 6.4.4.0 | 6.5.0   | 2.3.4 & 2.3.5  | 1.8  |
```

Comment:
you should remove the `1.0.0` line, I think we left `0.8.1` because this supports older versions

Reply:
+1, move everything to https://github.com/adobe/commerce-cif-connector/blob/master/VERSIONS.md

Reply:
Ah you just did :-)